.golangci.yml

```diff
@@ -41,6 +40,7 @@ linters-settings:
       - name: error-naming
       - name: exported
       - name: if-return
+      - name: import-shadowing
```

Comment:
We probably should enable it for the scaffolds as well.

Reply:
Great idea, let me check that out